spec/defines/vhost_spec.rb

```diff
@@ -1221,6 +1221,32 @@
           it { is_expected.not_to contain_concat__fragment('NameVirtualHost 127.0.0.1:8080') }
         end
 
+        describe 'serveraliases parameter' do
+          let :params do
+            {
+              'serveraliases' => serveraliases,
+            }
+          end
```

Comment:
```suggestion
          let :params do
            {
              'docroot'       => '/path/to/docroot',
              'serveraliases' => serveraliases,
            }
          end
```

should help according to the failures bellow.